app.py

```diff
@@ -119,13 +133,33 @@ def post(self):
         self.finish(json.dumps(finish_ret))
 
 
+def get_user_img_prefix(self):
+    cookie = self.get_cookie(app_cookie)
+    user_id = cookie if cookie else ""
+    return temp_img_prefix + user_id + "-"
+
+
+def valid_user_img(self, img):
+    default_img = not img.startswith(static_img_path + temp_img_prefix)
```

Comment:
Can you add a docstring here? There's some funkiness going on that's not easy to follow. Explaining what the method actually does would be helpful.

Reply:
will do